__tests__/app.test.js

```diff
@@ -12,7 +12,7 @@ afterAll(() => {
   return db.end;
 });
 
-describe.only("GET /api/topics", () => {
+describe("GET /api/topics", () => {
```

Comment:
I appreciate this was probably in response to feedback for another ticket but please keep each PR specific to the ticket the it's covering. Changes like this can go on their own branch and (unless otherwise stated in the feedback) can generally be merged in without submitting another PR.